### DISCUSSION TITLE
Compiler errors in Delphi 10.3.3

### DISCUSSION THREAD

**Discussion by george-kar** [SchneiderInfosystems/FB4D#148] [Q&A] [answered]
Hello, I am trying to build the examples provided in my Delphi 10.3.3 Rio and I got errors on 

FB4D.Helpers:
[dcc64 Error] FB4D.Helpers.pas(663): E2250 There is no overloaded version of 'SameText' that can be called with these arguments
and many other similar

then
[dcc64 Error] FB4D.Helpers.pas(714): E2010 Incompatible types: 'TRESTContentType' and 'string'

then
[dcc64 Error] FB4D.Helpers.pas(1152): E2010 Incompatible types: 'string' and 'Set'

Any idea?

**Comment by SchneiderInfosystems** [comment, marked as answer]
Oh unfortunately, Delphi 10.3.3 declares the TRESTContentType as enum. 
I also see other problems compiling with Delphi 10.3.3.
I have fixed all defects with De 10.3.3. Thank you for this report.

Thank you for any donation if you can use this library in a commercial project.

**Reply by george-kar** [reply]
If I can entirely use the component to my project I will donate. For the time I need the OnDisconnect class to fully work with it.

https://github.com/SchneiderInfosystems/FB4D/discussions/149
